apps/emqx_bridge/src/schema/emqx_bridge_enterprise.erl

```diff
@@ -235,11 +235,10 @@ mongodb_structs() ->
 
 kafka_structs() ->
     [
-        {kafka_producer,
+        {kafka,
```

Comment:
Note: I think changing this main type and adding an alias might have been a mistake for v1.  We had given up renaming it in v1 because of the problems of juggling raw configs around that didn't interact well with hocon alias, and the compatibility layer may have masked this issue before.

Reply:
this is shared schema with v2, does this mean v2 will continue to use `kafka` for producer ?

Reply:
Ok. Thanks. I had the impression it was shared.